kroxylicious-operator/src/test/java/io/kroxylicious/kubernetes/operator/KafkaProtocolFilterReconcilerIT.java

```diff
@@ -160,21 +150,16 @@ void shouldUpdateStatusOnConfigMapDelete() {
         assertStatusResolvedRefs(filterOne, Condition.Status.FALSE, "Referenced ConfigMaps [b] not found");
     }
 
-    private KafkaProtocolFilter filter(String filterName, String refA, String refB) {
-        // @formatter:off
-        return new KafkaProtocolFilterBuilder()
-                .withNewMetadata()
-                    .withName(filterName)
-                .endMetadata()
-                .withNewSpec()
-                    .withType("org.example.Filter")
+    private KafkaProtocolFilter filter(LocallyRunningOperatorRbacHandler.TestActor testActor, String refA, String refB) {
```

Comment:
`testActor` is already a field so personally I'd remove it from this methods args.